src/components/Button.tsx

```diff
@@ -89,6 +89,13 @@ const Outline = Button.extend`
 		getColor(props, 'color', 'main') || props.theme.colors.text.main};
 	background: ${props => props.color || 'none'};
 	border: 1px solid;
+
+	&:hover,
+	&:focus,
+	&:active {
+		background-color: ${props => props.bg ?
+			darken(props.bg as string) : (getColor(props, 'bg', 'dark') || props.theme.colors.tertiary.dark)};
```

Comment:
What's the case that this fallback to the tertiary color is used?

Reply:
This happens if you don't specify a primary / seconday or tertiary option. eg
```
<Button outline>button</Button>
```